spec/ameba/rule/style/verbose_block_spec.cr

```diff
@@ -89,9 +89,11 @@ module Ameba::Rule::Style
         rule = VerboseBlock.new
         rule
           .tap(&.exclude_prefix_operators = true)
+          .tap(&.exclude_operators = true)
           .catch(source).should be_valid
         rule
           .tap(&.exclude_prefix_operators = false)
+          .tap(&.exclude_operators = false)
```

Comment:
This is where I was a bit confused. **Without** the additional setting of `exclude_operators` to `false` (since the default is now `true`), this was returning all of the following as issues:

```crystal
source = Source.new %(
          (1..3).sum { |i| +i }
          (1..3).sum { |i| -i }
          (1..3).sum { |i| ~i }
        )
```

I tried to do some basic `puts` debugging in the `prefix_operator?` and `operator?` methods to determine whether this was intentional or not, but I'm not 100% sure.

Reply:
That's correct. `ExcludePrefixOperators` setting is additionally narrowing `ExcludeOperators`.

Reply:
Got it! I wonder if I should maybe add a comment after the docblock reference to the setting to add some verbiage saying as much more explicitly?